src/CMakeLists.txt

```diff
@@ -337,23 +338,10 @@ find_package(Qt5 5.7 REQUIRED COMPONENTS Core Multimedia Network OpenGL UiTools
 # This is not present before Qt 5.7:
 find_package(Qt5 COMPONENTS Gamepad QUIET)
 
-# This is not present before Qt 5.9:
```

Comment:
I thought we are still building against Qt 5.7 so handling Qt modules that may not (OS permitting) be available means this should be left in IMHO as should the comments following about which versions should have what - it may help when debugging why something doesn't work on a particular OS/Qt version as it all helps to document our expectations.